### PR TITLE
Fix benchmark scripts

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

### CODE REVIEW

File: src/tasks/batch_integration/resources_scripts/run_benchmark.sh [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+
+trace {
+    enabled = true
+    overwrite = true
+    file    = "$publish_dir/trace.txt"
+}
 HERE
 
+cat /tmp/nextflow.config
+
```
Comment (rcannood) [inline src/tasks/batch_integration/resources_scripts/run_benchmark.sh:26]:
```suggestion
```

File: src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml [hunk 2]
```diff
@@ -20,26 +20,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml:22]: Remove the example if there is a default

```suggestion
```

File: src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml [hunk 3]
```diff
@@ -20,26 +20,31 @@ functionality:
...
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml:28]: Remove the example if there is a default

```suggestion
```

File: src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml [hunk 4]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml:46]: Remove the example if there is a default

```suggestion
```

File: src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml [hunk 5]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml:40]: Remove the example if there is a default

```suggestion
```

File: src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml [hunk 6]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/batch_integration/workflows/run_benchmark/config.vsh.yaml:34]: Remove the example if there is a default

```suggestion
```

File: src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml [hunk 7]
```diff
@@ -20,26 +20,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml:22]: Remove the example if there is a default

```suggestion
```

File: src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml [hunk 8]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml:46]: Remove the example if there is a default

```suggestion
```

File: src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml [hunk 9]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml:40]: Remove the example if there is a default

```suggestion
```

File: src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml [hunk 10]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml:34]: Remove the example if there is a default

```suggestion
```

File: src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml [hunk 11]
```diff
@@ -20,26 +20,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/denoising/workflows/run_benchmark/config.vsh.yaml:28]: Remove the example if there is a default

```suggestion
```

File: src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml [hunk 12]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml:46]: Remove the example if there is a default

```suggestion
```

File: src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml [hunk 13]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml:40]: Remove the example if there is a default

```suggestion
```

File: src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml [hunk 14]
```diff
@@ -20,26 +20,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml:34]: Remove the example if there is a default

```suggestion
```

File: src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml [hunk 15]
```diff
@@ -20,26 +20,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml:28]: Remove the example if there is a default

```suggestion
```

File: src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml [hunk 16]
```diff
@@ -20,26 +20,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/dimensionality_reduction/workflows/run_benchmark/config.vsh.yaml:22]: Remove the example if there is a default

```suggestion
```

File: src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml [hunk 17]
```diff
@@ -27,26 +27,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml:53]: Remove the example if there is a default

```suggestion
```

File: src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml [hunk 18]
```diff
@@ -27,26 +27,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml:47]: Remove the example if there is a default

```suggestion
```

File: src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml [hunk 19]
```diff
@@ -27,26 +27,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml:41]: Remove the example if there is a default

```suggestion
```

File: src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml [hunk 20]
```diff
@@ -27,26 +27,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml:35]: Remove the example if there is a default

```suggestion
```

File: src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml [hunk 21]
```diff
@@ -27,26 +27,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/label_projection/workflows/run_benchmark/config.vsh.yaml:29]: Remove the example if there is a default

```suggestion
```

File: src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml [hunk 22]
```diff
@@ -28,26 +28,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml:30]: Remove the example if there is a default

```suggestion
```

File: src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml [hunk 23]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml:54]: Remove the example if there is a default

```suggestion
```

File: src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml [hunk 24]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml:48]: Remove the example if there is a default

```suggestion
```

File: src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml [hunk 25]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml:42]: Remove the example if there is a default

```suggestion
```

File: src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml [hunk 26]
```diff
@@ -28,26 +28,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/match_modalities/workflows/run_benchmark/config.vsh.yaml:36]: Remove the example if there is a default

```suggestion
```

File: src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml [hunk 27]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
+          default: dataset_uns.yaml
         - name: "--output_task_info"
           type: file
           required: true
           direction: output
           example: task_info.yaml
```
Comment (rcannood) [inline src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml:54]: Remove the example if there is a default

```suggestion
```

File: src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml [hunk 28]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
+          default: metric_configs.yaml
         - name: "--output_dataset_info"
           type: file
           required: true
           direction: output
           example: dataset_uns.yaml
```
Comment (rcannood) [inline src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml:48]: Remove the example if there is a default

```suggestion
```

File: src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml [hunk 29]
```diff
@@ -28,26 +28,31 @@ functionality:
...
           type: file
           required: true
           direction: output
           example: method_configs.yaml
+          default: method_configs.yaml
         - name: "--output_metric_configs"
           type: file
           required: true
           direction: output
           example: metric_configs.yaml
```
Comment (rcannood) [inline src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml:42]: Remove the example if there is a default

```suggestion
```

File: src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml [hunk 30]
```diff
@@ -28,26 +28,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
+          default: score_uns.yaml
         - name: "--output_method_configs"
           type: file
           required: true
           direction: output
           example: method_configs.yaml
```
Comment (rcannood) [inline src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml:36]: Remove the example if there is a default

```suggestion
```

File: src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml [hunk 31]
```diff
@@ -28,26 +28,31 @@ functionality:
           direction: output
           description: A yaml file containing the scores of each of the methods
           example: score_uns.yaml
```
Comment (rcannood) [inline src/tasks/predict_modality/workflows/run_benchmark/config.vsh.yaml:30]: Remove the example if there is a default

```suggestion
```

**Review by rcannood — APPROVED** [review]
Minor changes, otherwise LGTM!

**Review by KaiWaldrant — COMMENTED** [review]
implement remarks

**Pull request merged**